state-chain/custom-rpc/src/lib.rs

```diff
@@ -45,6 +46,38 @@ use std::{
 	sync::Arc,
 };
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct ScheduledSwap {
+	pub swap_id: SwapId,
+	pub base_asset: Asset,
+	pub quote_asset: Asset,
+	pub side: Side,
+	pub amount: NumberOrHex,
```

Comment:
Can you use U256 instead (same below), we're trying to remove NumberOrHex, it doesn't behave how you think that it does. The serialization changes depending on the size of the number. But U256 is always hex, but otherwise this is good to go imo.

Reply:
This is want the other rpcs do (use U256)

Reply:
> it doesn't behave how you think that it does

I did check that `u128` always encodes as hex before using this.

> This is what the other rpcs do (use U256)

I see plenty of rpcs using `NumberOrHex` for amounts. Hence, I wasn't sure if we wanted to use `NumberOrHex` or `U256`. Happy to change to `U256`.